src/SFML/Audio/SoundFileWriterOgg.cpp

```diff
@@ -63,7 +70,8 @@ bool SoundFileWriterOgg::open(const std::filesystem::path& filename, unsigned in
     m_channelCount = channelCount;
 
     // Initialize the ogg/vorbis stream
-    ogg_stream_init(&m_ogg, std::rand());
+    std::uniform_int_distribution serialNumberDistribution(0, RAND_MAX);
+    ogg_stream_init(&m_ogg, serialNumberDistribution(rng));
```

Comment:
```suggestion
    ogg_stream_init(&m_ogg, std::uniform_int_distribution{0, RAND_MAX}(rng()));
```

Reply:
What's the advantage of hiding the mt19937 within a function?